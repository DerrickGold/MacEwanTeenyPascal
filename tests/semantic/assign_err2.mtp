(*negative index constant*)
const two := 2;
var a,b,c : integer;
    ar1, ar2 : array(20) of integer;

begin

	ar2(-1) := ar1(two)

end.