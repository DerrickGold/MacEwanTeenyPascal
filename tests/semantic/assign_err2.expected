
Semantic Error [line 8]: 
	Index out of bounds: negative index '-1' unsupported.
